packages/create-vite/template-vue-ts/src/components/HelloWorld.vue

```diff
@@ -25,7 +25,7 @@ const count = ref(0)
   </p>
   <p>
     Install
-    <a href="https://github.com/johnsoncodehk/volar" target="_blank">Volar</a>
+    <a href="https://github.com/vuejs/language-tools" target="_blank">Volar</a>
```

Comment:
I've renamed it back to Volar to keep with the naming in `create-vue` for now. I think it's more recognizable too.